CSharp/Library/Microsoft.Bot.Connector.Shared/OAuthAPI/OAuthClient.cs

```diff
@@ -37,6 +37,11 @@ public partial class OAuthClient : ServiceClient<OAuthClient>, IOAuthClient
         /// </summary>
         public virtual IOAuthApi OAuthApi { get; private set; }
```

Comment:
Should we mark this obsolete?

Reply:
My thinking is that if at some point we do a cleanup that include breaking changes, removing obsolete stuff should leave nice abstractions. Seems like the rest of your PR is making the old IOauthApi obsolete, so if possible take it all the way, unless I'm missing something.

Reply:
Can do.

---
In reply to: [330821759](https://github.com/microsoft/BotBuilder-V3/pull/165#discussion_r330821759) [](ancestors = 330821759)